test/utils/init.js

```diff
@@ -20,6 +20,34 @@ chai.use((chaiAPI, utils) => {
       prettyDOM(document.activeElement),
     );
   });
+
+  chai.Assertion.addProperty('ariaHidden', function elementIsAccessible() {
+    const element = utils.flag(this, 'object');
+
+    // "An element is considered hidden if it, or any of its ancestors are not
+    // rendered or have their aria-hidden attribute value set to true."
+    // -- https://www.w3.org/TR/wai-aria-1.1/#aria-hidden
+    let previousNode = element;
+    let currentNode = element;
+    let ariaHidden = false;
+    while (
+      currentNode !== null &&
+      currentNode !== document.documentElement &&
```

Comment:
add as comment: stopping at the `html` for better debug output. Don't know a use case for apply aria attributes to the `html` element anyway.